### PR TITLE
[admin] Ports the AFK verb from prebase

### LINKED ISSUE #4629: Bring back the [AFK] command.
Just a nice QoL thing; that's all.

### PR DESCRIPTION
**Pull request by Altoids1**
## Sokath, his eyes covered!
image

Fixes #4629.

#### Changelog
:cl:  Altoids
rscadd: The AFK verb has been re-added! Use it when you want to quickly inform everyone of you being AFK.
/:cl:

### CODE REVIEW

File: yogstation/code/modules/client/verbs/afk.dm [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/client/verb/afk()
+	set name = "AFK"
+	set category = "OOC"
+	set desc = "Report to Admins and your peers that you will go AFK."
+
+	if(mob)
+		var/mob/M = mob
+
+		if(!M || !M.job)
```
Comment (Amelia0010) [inline yogstation/code/modules/client/verbs/afk.dm]:
```suggestion
		if(!M.job)
```
You already check whether there's a mob in line 6.

File: yogstation/code/modules/client/verbs/afk.dm [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+					text = "I need to shut my eyes for a short while."
+				else if(time == "15 minutes")
+					text = "I need to shut my eyes for a bit. Please keep an eye on me while I am resting."
+				else if(time == "30 minutes")
+					text = "I need to shut my eyes for quite a while. Please keep an eye on me while I am resting."
+				else if(time == "Whole round")
+					text = "I need to shut my eyes for a long time... Someone please take over my station responsibilities."
+
+			var/alert_admins = FALSE
+			if(M.job == "AI" || istype(M, /mob/living/silicon))
```
Comment (Amelia0010) [inline yogstation/code/modules/client/verbs/afk.dm]:
```suggestion
			if(istype(M, /mob/living/silicon))
```

File: yogstation/code/modules/client/verbs/afk.dm [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+				alert_admins = TRUE
+				M.say(".c [text] This is the [M.job] signing off for now.")
+				M.say(".o [text] This is the [M.job] signing off for now.")
+			else if(M.job == "Captain" || M.job == "Head of Personnel" || M.job == "Head of Security" || M.job == "Chief Engineer" || M.job == "Research Director" || M.job == "Chief Medical Officer")
```
Comment (Amelia0010) [inline yogstation/code/modules/client/verbs/afk.dm]:
```suggestion
			else if(M.job in GLOB.command_positions)
```

File: yogstation/code/modules/client/verbs/afk.dm [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+				alert_admins = TRUE
+				M.say(".h [text] This is the [M.job] signing off for now.")
```
Comment (Amelia0010) [inline yogstation/code/modules/client/verbs/afk.dm:53]: doesn't alert command?

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by Lynameep** [comment]
Thanks dad.

**Pull request merged**